components/prize_pool/wikis/overwatch/prize_pool_custom.lua

```diff
@@ -41,14 +41,6 @@ function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
 		Variables.varDefault('tournament_type')
 	)
 
-	local participantLower = mw.ustring.lower(lpdbData.participant)
-
-	Variables.varDefine(participantLower .. '_prizepoints', lpdbData.extradata.prizepoints)
```

Comment:
Thought you wanted the prizepoints based on PR title?

Reply:
Oops, you're right, I removed too much